pages/app/components/theme-switcher.tsx

```diff
@@ -26,44 +26,72 @@ export default function ThemeSwitcher() {
   }
 
   return (
-    <SpaceBetween direction="horizontal" size="xs">
-      <label>
-        Theme
-        <select defaultValue={THEME}>
-          <option value={THEME}>{THEME}</option>
-        </select>
-      </label>
-      <label>
-        <input {...vrSwitchProps} />
-        Visual refresh
-      </label>
-      <label>
-        <input
-          id="mode-toggle"
-          type="checkbox"
-          checked={mode === 'dark'}
-          onChange={event => setMode(event.target.checked ? Mode.Dark : Mode.Light)}
-        />
-        Dark mode
-      </label>
-      <label>
-        <input
-          id="density-toggle"
-          type="checkbox"
-          checked={urlParams.density === 'compact'}
-          onChange={event => setUrlParams({ density: event.target.checked ? Density.Compact : Density.Comfortable })}
-        />
-        Compact mode
-      </label>
-      <label>
-        <input
-          id="disabled-motion-toggle"
-          type="checkbox"
-          checked={urlParams.motionDisabled}
-          onChange={event => setUrlParams({ motionDisabled: event.target.checked })}
-        />
-        Disable motion
-      </label>
-    </SpaceBetween>
+    <Popover
+      position="bottom"
+      size="large"
+      content={
+        <SpaceBetween direction="vertical" size="xs">
+          <label>
+            Theme
+            <select defaultValue={THEME}>
+              <option value={THEME}>{THEME}</option>
+            </select>
+          </label>
+
+          <label>
+            Direction
+            <select
+              defaultValue={urlParams.direction}
+              onChange={event => {
+                setUrlParams({ direction: event.target.value as 'rtl' | 'ltr' });
+                window.location.reload();
```

Comment:
Why do we need to reload?

Reply:
There will be a subset of components (Dropdown, Popover) that will need to detect directionality at runtime and apply styles through the CSSOM. If one of these components is in an open state and the document direction is changed they will not respond dynamically without a mutation observer. This is something of an edge case that we can revisit, but after discussion with @just-boris we decided to proceed with the same approach as Visual Refresh to start.